yarn.lock

```diff
@@ -2038,6 +2038,15 @@
   dependencies:
     "@loaders.gl/loader-utils" "3.0.8"
 
+"@loaders.gl/loader-utils@3.0.12":
```

Comment:
You have multiple versions of loaders installed

Reply:
I'm at a loss on how to get yarn to do what I want here. I've updated `package.json` to pick a specific version of wkt (`"@loaders.gl/wkt": "3.0.8"`) and then there is only one version of loaders. But when I set it to `^3.0.8` like the other packages, yarn insists on pulling in 3.0.12 in some places.

Reply:
This is now no longer relevant. Please ignore above comment